README.md

```diff
@@ -360,6 +361,39 @@ permalink.stringify({year: '2014', month: '01', day: '31', title: 'test'})
 // 2014/01/31/test
 ```
 
+### prettyUrls(url, [options])
+
+Rewrite urls to pretty URLs.
+
+Option | Description | Default
+--- | --- | ---
+`trailing_index` | `/about/index.html -> /about/` when `false` | `true`
+`trailing_html` | `/about.html -> /about` when `false` | `true`
```

Comment:
Since `trailing_index` retain trailing slash by default, I think `trailing_html` should retain it as well. If a trailing slash is not desired in _both_ `trailing_index` and `trailing_html`, can disable `trailing_slash` (suggest to introduce this option in this PR too).

Reply:
For most web server (like nginx, caddy), when they receive a URL like `/foo/bar/`, they will looks up html files (`index.html`, `index.htm`, `index.php`, `index.asp`, etc.) under `foo/bar/` directory. But when receive a URL like `foo/bar`, they will look up `bar.html`, `bar.htm` under `/foo/` directory.

As you can see, both `https://hexo.io/docs/permalinks` and `https://hexo.io/docs/permalinks.html` will return `200 OK` while `https://hexo.io/docs/permalinks/` will be 301 redirect to `https://hexo.io/docs/permalinks`.

Reply:
you're right, I mistook netlify's pretty url. Upon checking,

`Rewrite link URLs to pretty URLs (/about.html -> /about, /about/index.html -> /about/)`